cloudinit/config/cc_mounts.py

```diff
@@ -317,6 +317,13 @@ def create_swap(fname, size, method):
 
     fstype = util.get_mount_info(swap_dir)[1]
 
+    if fstype == "btrfs":
+        try:
+            subp.subp(["truncate", "-s", "0", fname])
```

Comment:
if this is the command that fails

Reply:
Is there any suggestion?

Reply:
if truncate fails, the log message will say failed to disable copy on write.

despite the order in which i made the comments, they show up in the order of line numbers

Reply:
So we should move first command outside the try block. Or add another try-catch for it?
The first command is likely won't be failed. It just create a new file.

Reply:
yes. if `truncate` fails, you don't want to catch the exception, because something is very wrong.